src/test/java/org/apache/commons/io/output/ByteArrayOutputStreamTest.java

```diff
@@ -350,6 +358,22 @@ public void testWriteZero(final String baosName, final BAOSFactory<?> baosFactor
         }
     }
 
+    @Test
+    public void testToByteArrayImplAndResetImpl() throws Exception {
+        final int file_size = (1024 * 4) + 1;
```

Comment:
- No underscore in local variable names. 
- No need for spelling out 4097 in an expression.